docs/access_and_permissions.md

```diff
@@ -302,13 +412,15 @@ flowchart TD
 
 ### Controllers
 
-| Class                                                                                  | Responsibility                                                                     |
-|----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|
-| [Account::ApplicationsController](/app/controllers/account/applications_controller.rb) | Managing own access to apps and accessing own permissions routes                   |
-| [Account::PermissionsController](/app/controllers/account/permissions_controller.rb)   | Managing own permissions                                                           |
-| [InvitationsController](/app/controllers/invitations_controller.rb)                    | Giving users access to apps and permissions while creating their account           |
-| [Users::ApplicationsController](/app/controllers/users/applications_controller.rb)     | Managing other users' access to apps and accessing other users' permissions routes |
-| [Users::PermissionsController](/app/controllers/users/permissions_controller.rb)       | Managing other users' permissions                                                  |
+| Class                                                                                             | Responsibility                                                                     |
+| ------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| [Account::ApplicationsController](/app/controllers/account/applications_controller.rb)            | Managing own access to apps and accessing own permissions routes                   |
```

Comment:
I wonder if this (and the users equivalent) actually needs updating to reflect that it doesn't have any destructive responsibilities, but rather presents views that provide access to the permissions controller actions? Or something to that effect

Reply:
Good point - I've had a go at this, hope it's clear enough

Reply:
Looks good to me!

Reply:
Looks good to me!